src/Uno.UI.RemoteControl.Messaging/IRemoteControlServer.cs

```diff
@@ -9,5 +10,7 @@ public interface IRemoteControlServer
 		string GetServerConfiguration(string key);
 
 		Task SendFrame(IMessage fileReload);
+
+		Task SendMessageToIDEAsync(IdeMessage message);
```

Comment:
Not sure to understand why we have 2 methods here, can the IDA channel replace the dev-server channel? More doc would be welcome :)